.circleci/config.yml

```diff
@@ -77,6 +135,24 @@ jobs:
             - store_artifacts:
                   path: ~/transformers/output.txt
                   destination: test_output.txt
+    run_examples_tpu:
```

Comment:
@zcain117 you need to add it to the workflow on line 219

Reply:
Great find, thank you!